cmd/release-controller/sync_verify_prow.go

```diff
@@ -152,18 +152,39 @@ func addReleaseEnvToProwJobSpec(spec *prowjobv1.ProwJobSpec, release *Release, m
 			}
 		}
 		if !hasReleaseImage {
-			c.Env = append(c.Env, corev1.EnvVar{Name: "RELEASE_IMAGE_LATEST", Value: release.Target.Status.PublicDockerImageRepository + ":" + releaseTag.Name})
+			switch architecture {
+			case "s390x":
```

Comment:
Can we add arm arch as well?

Reply:
Sure, I can add it.  I just need to know what you named the release.

Reply:
Its not in progress we can add a name and then use it for arm.
Something similar to `arm-initial` & `arm-latest`

Reply:
Ok, I'll use that for now.

Reply:
Changes pushed